app/controllers/admin_user_controller.rb

```diff
@@ -159,7 +159,7 @@ def changed_role_ids
   def check_role_requirements
     role_names = Role.
                    where(:id => params[:admin_user][:role_ids]).
-                     pluck(:name).map{ |role| role.to_sym }
+                     pluck(:name).to_a.map{ |role| role.to_sym }
```

Comment:
short answer, yes `map` is a problem where an `ActiveRecord_Relation` is being returned ...

```
reqs = InfoRequest.all
reqs.map
=> NoMethodError: undefined method `map' for InfoRequest::ActiveRecord_Relation:Class
```

*but* I've also called mistakenly called `.to_a` on Hashes and Arrays. Fixing...